presets/ui-kit/src/index.ts

```diff
@@ -51,6 +51,9 @@ export {
   useDataTableSortingState,
 } from '@commercetools-uikit/hooks';
 
-export { customProperties } from '@commercetools-uikit/design-system';
+export {
+  customProperties,
```

Comment:
Perhaps we can also add the deprecated comment here.
```suggestion
  /** @deprecated use `designTokens` instead */
  customProperties,
```